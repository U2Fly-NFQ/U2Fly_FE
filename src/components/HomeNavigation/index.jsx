const HomeNavigation = () => {
  return <div>Home navigation</div>
}

export default HomeNavigation
