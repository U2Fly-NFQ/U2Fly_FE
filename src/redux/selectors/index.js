export * from './auth'
export * from './bookingFlightSelector'
export * from './ticket'
