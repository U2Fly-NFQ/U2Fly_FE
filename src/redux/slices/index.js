export * from './searchSlice'
export * from './filterSlice'
export * from './flightSlice'
