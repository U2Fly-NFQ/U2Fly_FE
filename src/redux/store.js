import { configureStore } from '@reduxjs/toolkit'
import bookingFlightsSlice from './slices/bookingFlightsSlice'

// import loginSlice from './slices/loginSlice'
import flightSlice from './slices/flightSlice'
import airportSlice from './slices/airportSlice'

const store = configureStore({
  reducer: {
    // login: loginSlice.reducer,
    bookingFlight: bookingFlightsSlice.reducer,
    flights: flightSlice.reducer,
    airports: airportSlice.reducer,
  },
})

export default store
