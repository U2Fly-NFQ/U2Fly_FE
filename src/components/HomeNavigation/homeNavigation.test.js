import { I18nextProvider } from 'react-i18next'
import i18n from '../../translations'
import React, { Suspense } from 'react'
import { store } from '../../redux/store'
import { Provider } from 'react-redux'
import { BrowserRouter as Router } from 'react-router-dom'

import HomeNavigation from './'
import { ConfigProvider } from 'antd'
import renderer from 'react-test-renderer'

describe('Home navigation', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <I18nextProvider i18n={i18n}>
          <Provider store={store}>
            <ConfigProvider>
              <Router>
                <Suspense>
                  <HomeNavigation />
                </Suspense>
              </Router>
            </ConfigProvider>
          </Provider>
        </I18nextProvider>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
