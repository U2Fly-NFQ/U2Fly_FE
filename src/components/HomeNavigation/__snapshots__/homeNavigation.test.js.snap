// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home navigation renders correctly 1`] = `
<div>
  Home navigation
</div>
`;
